ftl/common/ftl_util.py

```diff
@@ -28,10 +28,7 @@ def AppendLayersIntoImage(lyr_imgs):
     with Timing('Stitching layers into final image'):
         for i, lyr_img in enumerate(lyr_imgs):
             if i == 0:
-                try:
-                    result_image = lyr_img.GetImage()
-                except AttributeError:
-                    result_image = lyr_img
+                result_image = lyr_img
```

Comment:
I think you want to change the line below as well? Looks like instead of passing the builders to this method you're now passing the images produced by the builders, so you shouldn't have to call `GetImage()` anymore.

`img = lyr_img.GetImage()` -> `img = lyr_img`

Reply:
yea, I meant to delete that line, it's changed now.  thanks